plugins/node/opentelemetry-instrumentation-memcached/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@opentelemetry/instrumentation-memcached",
+  "version": "0.21.0",
+  "description": "OpenTelemetry memcached automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "compile": "npm run version:update && tsc -p .",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "prepare": "npm run compile",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "test:debug": "cross-env RUN_MEMCACHED_TESTS_LOCAL=true ts-mocha --inspect-brk --no-timeouts -p tsconfig.json 'test/**/*.test.ts'",
+    "test:local": "cross-env RUN_MEMCACHED_TESTS_LOCAL=true npm run test",
+    "version:update": "node ../../../scripts/version-update.js"
+  },
+  "keywords": [
+    "opentelemetry",
+    "memcached",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^0.21.0"
+  },
+  "devDependencies": {
+    "@opentelemetry/api": "0.21.0",
```

Comment:
you could use 1.0 directly (pr is already opened for contrib https://github.com/open-telemetry/opentelemetry-js-contrib/pull/540)